suricata/update/main.py

```diff
@@ -1264,6 +1267,12 @@ def _main():
         logger.info("Loading %s.", drop_conf_filename)
         drop_filters += load_drop_filters(drop_conf_filename)
 
+    # Load user provided output rules filename
+    rule_filename=config.get("output-rule-filename")
```

Comment:
We could use a little [PEP8](https://pep8.org/) magic here to make it look cleaner. You could probably setup your editor to do that for you or use `pep8` and `autopep8` utilities.